zh_cn/deco.metallslab.json

```diff
@@ -1,37 +0,0 @@
-			"8": [{"name":"futurepack:erz_blocke", "meta":4},{"name":"futurepack:color_iron", "meta":0},{"name":"futurepack:color_iron", "meta":1},{"name":"futurepack:color_iron", "meta":2},{"name":"futurepack:color_iron", "meta":3},{"name":"futurepack:color_iron", "meta":4},{"name":"futurepack:color_iron", "meta":5},{"name":"futurepack:color_iron", "meta":6},{"name":"futurepack:color_iron", "meta":7},{"name":"futurepack:color_iron", "meta":8},{"name":"futurepack:color_iron", "meta":9},{"name":"futurepack:color_iron", "meta":10},{"name":"futurepack:color_iron", "meta":11},{"name":"futurepack:color_iron", "meta":12},{"name":"futurepack:color_iron", "meta":13},{"name":"futurepack:color_iron", "meta":14},{"name":"futurepack:color_iron", "meta":15}],
-			"9": [{"name":"futurepack:erz_blocke", "meta":4},{"name":"futurepack:color_iron", "meta":0},{"name":"futurepack:color_iron", "meta":1},{"name":"futurepack:color_iron", "meta":2},{"name":"futurepack:color_iron", "meta":3},{"name":"futurepack:color_iron", "meta":4},{"name":"futurepack:color_iron", "meta":5},{"name":"futurepack:color_iron", "meta":6},{"name":"futurepack:color_iron", "meta":7},{"name":"futurepack:color_iron", "meta":8},{"name":"futurepack:color_iron", "meta":9},{"name":"futurepack:color_iron", "meta":10},{"name":"futurepack:color_iron", "meta":11},{"name":"futurepack:color_iron", "meta":12},{"name":"futurepack:color_iron", "meta":13},{"name":"futurepack:color_iron", "meta":14},{"name":"futurepack:color_iron", "meta":15}],
-			"out": [{"name":"futurepack:metal_slap0", "meta":0, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":0, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":1, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":2, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":3, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":4, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":5, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":6, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":7, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":8, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":9, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":10, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":11, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":12, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":13, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":14, "size":6},{"name":"futurepack:color_iron_step_half_0", "meta":15, "size":6}]
-		}
-	},
-	"信息：可使用喷枪进行染色",
-	"信息：放置在一起的两个台阶破坏时会掉落一个完整的方块。"
-]
```

Comment:
why was this file deleted?

Reply:
They have no practical effect.
There is no corresponding file in folder [en_us](https://github.com/Wugand/FuturePack-Language/tree/master/en_us).
This recipe is repeated with [deco.metall.json](https://github.com/Wugand/FuturePack-Language/blob/5cd98289a5107e4f97287449126d9d33e387225f/zh_cn/deco.metall.json#L5)（`	{"link":"futurepack:lang/part/cr_deco.metallslab1.json"},` ）.